src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -116,6 +120,32 @@ public void addInterview(Interview i) {
         interviews.add(i);
     }
 
+    //// position-level operations
+
+    /**
+     * Returns true if a position that is the same as {@code position} exists in the address book.
+     */
+    public boolean hasPosition(Position position) {
+        requireNonNull(position);
+        return positions.contains(position);
```

Comment:
Minor nitpick here but `position` and `positions` are quite similar. Would it be better to name the parameter `p` instead?